### PR TITLE
didactic changes

### PR DESCRIPTION
**Pull request by Mirabellensaft**
last batch of my edits! (of the beginner book)

### CODE REVIEW

File: beginner/apps/src/bin/radio-recv.rs [hunk 1]
```diff
@@ -17,6 +17,8 @@ fn main() -> ! {
     let mut radio = board.radio;
     let mut timer = board.timer;
 
+    radio.set_channel(Channel::_20);
```
Comment (Lotterleben) [inline beginner/apps/src/bin/radio-recv.rs:20]: do we need a `// TODO change the channel value if you've changed the channel used by the dongle` comment here?

File: embedded-workshop-book/src/SUMMARY.md [hunk 2]
```diff
@@ -20,6 +20,7 @@
         - [Link Quality Indicator (LQI)](./link-quality.md)
     - [Radio In ](./radio-in.md)
     - [Radio Puzzle](./radio-puzzle.md)
+    - [Radio Puzzele Help](./radio-puzzle-help.md)
```
Comment (japaric) [inline embedded-workshop-book/src/SUMMARY.md:23]:
```suggestion
    - [Radio Puzzle Help](./radio-puzzle-help.md)
```

File: embedded-workshop-book/src/radio-puzzle-help.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    let mut my_map = FnvIndexMap::<_, _, U16>::new();
+    my_map.insert(b'A', b'~').unwrap();
+
+    // A vector with a fixed capacity of 8 elements allocated on the stack
+    let mut my_vec = Vec::<_, U8>::new();
+    my_vec.push(b'A').unwrap();
+}
+```
+
+If you haven't used a stack-allocated collection before note that you'll need to specify the capacity of the collection as a type parameter using one of the "type-level values" in the `heapless::consts` module. The [`heapless::IndexMap` documentation][indexMap] of the `heapless` crate has some usage examples, as does the [`heapless::Vec` documentation][vec].
```
Comment (Lotterleben) [inline embedded-workshop-book/src/radio-puzzle-help.md:29]: I remember that it took me a while to understand that `U16` is a `heapless` constant, not a Rust quirk– would it make sense to clarify this by writing something like "...type parameter using one of the "type-level values" in the `heapless::consts` module (e.g. `U8`, `U64` etc)."?

File: embedded-workshop-book/src/radio-puzzle.md [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
 
 Like in the previous sections the Dongle will listen for radio packets -- this time over *channel 25* -- while also logging messages over a USB/serial interface.
 
-Open the `beginner/apps` folder in VS Code; then open the `src/bin/radio-puzzle.rs` file.
+✅ Open the `beginner/apps` folder in VS Code; then open the `src/bin/radio-puzzle.rs` file. Run the program. 
 
-Your task in this section is to decrypt the [substitution cipher] encrypted *ASCII* string stored in the Dongle. The string has been encrypted using *simple substitution*.
 
-[substitution cipher]: https://en.wikipedia.org/wiki/Substitution_cipher
+## Dongle Responses
```
Comment (Lotterleben) [inline embedded-workshop-book/src/radio-puzzle.md:17]: should we add a note taht the hlp chapter exists somewhere in this file?

File: beginner/apps/src/bin/radio-recv.rs (hunk 1 above)
Comment (japaric) [reply beginner/apps/src/bin/radio-recv.rs:20]: a note would be good to have, yes "20 is the default channel of loopback.hex; you may need to change this if ran `change-channel`"

File: embedded-workshop-book/src/radio-puzzle-help.md (hunk 3 above)
Comment (japaric) [reply embedded-workshop-book/src/radio-puzzle-help.md:29]: doesn't hurt to add a small list of examples, imo

File: beginner/apps/src/bin/radio-recv.rs (hunk 1 above)
Comment (Mirabellensaft) [reply beginner/apps/src/bin/radio-recv.rs:20]: There is a reminder in the text

Comment (Mirabellensaft) [reply beginner/apps/src/bin/radio-recv.rs:20]: I added a reminder to the book

Comment (Lotterleben) [reply beginner/apps/src/bin/radio-recv.rs:20]: yup, wasn't sure if we needed a "FYI this is the place that we meant" hint but you're right, it's pretty easy to spot

File: embedded-workshop-book/src/radio-puzzle-help.md (hunk 3 above)
Comment (Mirabellensaft) [reply embedded-workshop-book/src/radio-puzzle-help.md:29]: Lotte, do you have time to do that?

Comment (Lotterleben) [reply embedded-workshop-book/src/radio-puzzle-help.md:29]: sure, 1sec

**Pull request merged**